release notes/v0.56.0.md

```diff
@@ -0,0 +1,67 @@
+- [browser#1547](https://github.com/grafana/xk6-browser/pull/1547) Fixes a panic due to events associated to stale frames.
+- [browser#1552](https://github.com/grafana/xk6-browser/pull/1552) Fixes a panic for `locator.selectOption` when value is an object.
+- [browser#1559](https://github.com/grafana/xk6-browser/pull/1559) Fixes a panic for `page.screenshot`.
+- [browser#1544](https://github.com/grafana/xk6-browser/pull/1544) Fixes a nil pointer dereference when calling `evaluate` or `evaluateHandle` with an invalid page function.
+- [#4058](https://github.com/grafana/k6/pull/4058) Fixes the namespaced object export when default is the only one available.
+
+## Maintenance and internal improvements
+
+- [browser#1504](https://github.com/grafana/xk6-browser/pull/1504), [browser#1510](https://github.com/grafana/xk6-browser/pull/1510), [browser#1511](https://github.com/grafana/xk6-browser/pull/1511) [browser#1512](https://github.com/grafana/xk6-browser/pull/1512) [browser#1513](https://github.com/grafana/xk6-browser/pull/1513), [browser#1518](https://github.com/grafana/xk6-browser/pull/1518), [browser#1519](https://github.com/grafana/xk6-browser/pull/1519), [browser#1520](https://github.com/grafana/xk6-browser/pull/1520), [browser#1523](https://github.com/grafana/xk6-browser/pull/1523), [browser#1521](https://github.com/grafana/xk6-browser/pull/1521), [browser#1524](https://github.com/grafana/xk6-browser/pull/1524), [browser#1526](https://github.com/grafana/xk6-browser/pull/1526), [browser#1527](https://github.com/grafana/xk6-browser/pull/1527), [browser#1531](https://github.com/grafana/xk6-browser/pull/1531),  Removes Sobek out of business logic layers.
+- [browser#1528](https://github.com/grafana/xk6-browser/pull/1528), [browser#1529](https://github.com/grafana/xk6-browser/pull/1529) Aligns repo with k6.
+- [browser#1517](https://github.com/grafana/xk6-browser/pull/1517) Fixes internal tests.
+- [browser#1535](https://github.com/grafana/xk6-browser/pull/1535), [browser#1537](https://github.com/grafana/xk6-browser/pull/1537), [#4137](https://github.com/grafana/k6/pull/4137), [#4148](https://github.com/grafana/k6/pull/4148) Updates dependencies.
+- [browser#1540](https://github.com/grafana/xk6-browser/pull/1540) Refactors internals.
+- [#4057](https://github.com/grafana/k6/pull/4057) Update k6-taskqueue-lib to drop goja dependency.
+- [#4076](https://github.com/grafana/k6/pull/4076) modules.NewModuleInstance doc updates.
+- [#4095](https://github.com/grafana/k6/pull/4095) Golangci-lint updated to v1.62.2.
+- [#4099](https://github.com/grafana/k6/pull/4099), [#4114](https://github.com/grafana/k6/pull/4114) Fixes to CI after browser merge, around running tests on windows.
+- [#4102](https://github.com/grafana/k6/pull/4102) Fix windows-packaging CI after update of the windows github action.
+
+
+## _Optional_ Roadmap
+
+_Discussion of future plans_
+
```

Comment:
```suggestion
```

Maybe we should mention some deprecation ideas?

Reply:
We should probably mention that we will change the end of test summary format, which is kind of a deprecation, in a sense. what do you think @joanlopez ?

Reply:
I've added the removal note for `k6/experimental/browser`.

Reply:
(By the way: please do not consider my comment blocking, and feel free to move forward with it)